lib/sufia/engine.rb

```diff
@@ -1,4 +1,12 @@
 module Sufia
+  class Statefile
+    def self.default
+      return '/tmp/minter-state' if Rails.env.development? || Rails.env.test?
+      raise NotImplementedError, "can't use the default statefile for production" unless Dir.exist?('/var/sufia')
```

Comment:
Are you certain about that? I thought it always ran the code in engine.rb, then ran your configuration over the top of that.